### DISCUSSION TITLE
Input Filter

### DISCUSSION THREAD

**Discussion by OvalMedia** [filamentphp/filament#4730] [Help] [answered]
I wanted to set up an input filter for a field (invoice numbers) but to my surprise there is no such thing. Or am I missing something?
I know there is the global search but that does not persist in the session (or is there a way?), a feature  which I desire.

So...

1. Do input filters exist, if so, how do I use them?
2. Can I make the global search persist in the session?
Thx

**Comment by danharrin** [comment, marked as answer]
You can use any form compnoent inside a filter like this - https://filamentphp.com/docs/2.x/tables/filters#custom-filter-forms

Alternatively, you can put a dedicated search input just above the invoice number field if you wish - 
https://filamentphp.com/docs/2.x/tables/columns/getting-started#searching-individually

**Reply by OvalMedia** [reply]
That's what I needed. Thank you.
